### ISSUE TITLE
Request - Full Disk Encryption

### ISSUE THREAD

**Issue by dofishswim** [Linux-for-Fydetab-Duo/Issues#1]
Hi @fydepanda ,

It would be great to have Full Disk Encyption as an option, it is definitively out of my knowledge personall but wondering if it would be possible to boot an installer via UEFI :)

Many thanks in advance,
Peace,
Tera

PS: thanks a million for bringing Linux to the FydeTab, it works great so far (besides the Stylus which was my primary push to buy the tablet)!! 👍

**Comment by fydepanda** [comment]
Sorry for the late response
> It would be great to have Full Disk Encyption as an option

You can definitely set it up yourself by following something like this  https://unix.stackexchange.com/questions/444931/is-there-a-way-to-encrypt-disk-without-formatting-it

> it is definitively out of my knowledge personall but wondering if it would be possible to boot an installer via UEFI

Technically it should be possible but UEFI doesnt have drivers for the FUSB302 (the type C) so that means you cant easily connect a Usb drive with an iso, then theres a different issue that is the OS you are booting needs to have support for the fydetab

**Comment by drippythesimpy** [comment]
> Sorry for the late response
> 
> > It would be great to have Full Disk Encyption as an option
> 
> You can definitely set it up yourself by following something like this https://unix.stackexchange.com/questions/444931/is-there-a-way-to-encrypt-disk-without-formatting-it
> 
> > it is definitively out of my knowledge personall but wondering if it would be possible to boot an installer via UEFI
> 
> Technically it should be possible but UEFI doesnt have drivers for the FUSB302 (the type C) so that means you cant easily connect a Usb drive with an iso, then theres a different issue that is the OS you are booting needs to have support for the fydetab

Yes but how do you modify the bootloader to decrypt root at startup? For BredOS all I see is /EFI/BOOT/BOOTAA64.EFI. Where is initrid stored? I tried modifying /etc/cryptab, /etc/fstab, and /etc/default/grub but wasn't able to get the bootloader to decrypt at boot.
